src/util/search.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * SelectMultipleFilter needs to parse values from format
+ * "has_all:a,b,c,d" or "a,b,c,d"
+ */
+export const parseSelectFilterOptions = uriComponentValue => {
+  return uriComponentValue && uriComponentValue.indexOf(':') >= 0
+    ? uriComponentValue.split(':')[1].split(',')
+    : uriComponentValue && uriComponentValue.indexOf(':') < 0
+    ? uriComponentValue.split(',')
+    : [];
+};
+
+const flatten = (acc, val) => acc.concat(val);
```

Comment:
I think `append` would be a more correct name for this function. It's not flattening a multi-dimensional array into a one-dimensional array as the name implies.